install/linux/usr/share/odemis/sim/mimas-sim.odm.yaml

```diff
@@ -151,6 +151,7 @@ MIMAS-Sim: {
         inverted: ['y'],  # TODO Need to check that the rotations go counter-clockwise
     },
     metadata: {
+        ION_BEAM_TO_SAMPLE_ANGLE: 0.12217,
```

Comment:
@pieleric What is the correct value to add here?

Reply:
I already have a PR (#2332) which adds this metadata. In general, any value between 0 and 90° could be correct. On the current MIMAS, it's 8°.